cmd/minishift/cmd/start_test.go

```diff
@@ -413,8 +413,11 @@ func Test_getslice_withCommandLine(t *testing.T) {
 	viper.Set("foo", []string{"hello", "world"})
 	defer viper.Reset()
 
-	if getSlice("foo") != nil {
-		t.Errorf("expected %+v. Got %+v", nil, getSlice("foo"))
+	expectedSlice := []string{"hello", "world"}
+	for i, v := range getSlice("bar") {
```

Comment:
There is a bug here. You are not testing anything here. You call _getSlice_ for _bar_. This will be empty and hence it will not go into the loop and pass the test. You need to add an asserts that you get a non empty slice. And you need of course to request 'foo', not 'bar'